libs/mcp-chat/src/lib/mcp-chat.component.scss

```
@@ -0,0 +1,126 @@
+    align-items: center;
+    gap: 6px;
+    font-size: 12px;
+    text-transform: uppercase;
+    opacity: 0.7;
+    margin-bottom: 4px;
+  }
+
+  &__message-text {
+    margin: 0;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)